src/pages/studyView/StudyViewPage.tsx

```diff
@@ -220,6 +222,30 @@ export default class StudyViewPage extends React.Component<
         this.showBookmarkModal = !this.showBookmarkModal;
     }
 
+    @observable showShareBookmarkModal = false;
```

Comment:
maybe we just call it "shareLinkModal"?    i think the idea of sharing encompasses bookmark.